inc/file.class.php

```diff
@@ -36,20 +36,12 @@
 /**
  * @since 0.1.30
  */
-class PluginFlyvemdmFile extends CommonDBTM {
-
-   // name of the right in DB
-   static $rightname = 'flyvemdm:file';
-
-   /**
-    * @var bool $usenotepad enable notepad for the itemtype (GLPi < 0.85)
```

Comment:
here and a few lines below you have the exact informations I talked about in the beginning of the review. The variable for GLPI < 0.85 may be safely dropped.